modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20257,6 +20262,8 @@ use-the-form-below-to-move-a-page-to-another-node=Use the form below to move a p
 use-the-form-below-to-rename-a-page,-moving-all-of-its-history-to-the-new-name=Use the form below to rename a page, moving all of its history to the new name. The old title will become a redirect page to the new title. Links to the old page title will not be changed.
 use-the-inherited-theme=Use the inherited theme.
 use-the-inherited-theme-x=Use the inherited theme. {0}
+use-the-javascript-url-field=Use the 'JavaScript URL' field.
+use-the-javascript-url-field-instead-of-the-attribute-src=Use the 'JavaScript URL' field instead of the attribute 'src'.
```

Comment:
Do we really need both of these keys. It seems that `Use the 'JavaScript URL' field instead of the attribute 'src'.` is much clearer than just `Use the 'JavaScript URL' field.`. Can we always use `Use the 'JavaScript URL' field instead of the attribute 'src'.`? @drakonux ?

Reply:
https://www.figma.com/file/qyFWx04eAd3tSnv2jzYDIL/lps-203536-add-JS-client-extensions-with-element-attributes?type=design&node-id=0-1&mode=design&t=fPxiSiKOv3R2CXo0-0

Reply:
@drakonux, this is the visuals of Evans's suggestion. What do you think? 

![Screenshot_20240304_143757](https://github.com/liferay-platform-experience/liferay-portal/assets/47724428/7952d14f-a97b-488f-b38b-95195330bb90)

Reply:
Thanks @drakonux

Reply:
That explanation makes sense to me. I'm fine with keeping it as the original design showed. Thank you.

Reply:
Thanks @ethib137!